src/Apps/Order/Components/OfferSummaryItem.tsx

```diff
@@ -21,6 +23,17 @@ const OfferSummaryItem = ({
       <Sans size="2" color="black60">
         List price: {order.totalListPrice}
       </Sans>
+      {offerNote && (
+        <>
+          <Spacer mb={[2, 3]} />
+          <Serif size={["2", "3t"]} weight="semibold" color="black100">
```

Comment:
Thanks again @pepopowitz for adding responsive typography sizes 🎉  -- changes the whole game